nucypher/blockchain/eth/sol/source/contracts/WorkLock.sol

```diff
@@ -338,11 +338,22 @@ contract WorkLock {
     }
 
     /**
-    * @notice Cancels distribution, makes possible to retrieve all bids and creator gets all tokens
+    * @notice Cancels distribution, makes possible to retrieve all bids and owner gets all tokens
     */
-    function shutdown() internal {
+    function shutdown() external onlyOwner {
+        require(!isClaimingAvailable(), "Claiming has already been enabled");
```

Comment:
s the any mitigation or follow up for this condition?

Reply:
Why need any mitigations?